system/system.proto

```diff
@@ -305,7 +285,7 @@ message Package {
   // For system packages this will take effect after reboot.
   bool activate = 5;
```

Comment:
I still wonder if `activate` is the right name for this field -- to me a true value here could be interpreted as "make this package active now".  does something like `requires_reboot` make sense ?

Reply:
I am open to a new name, just not sure what that might be :)

For more context, when you transfer a file, the file can be stored on the device, but not applied. So after transferring a file (without activation) a reboot of the device would not set the package to active. You have to specify somehow to set a specific package active.

Reply:
I think I'm getting hung up on the "reboot" part of this -- maybe we can change the comment to say something like:
```
Indicates that the package should be made active after receipt on the device.
For system image packages, the new image is expected to be active after a reboot.
```

Reply:
Sure, done.